spectral_cube/lower_dimensional_structures.py

```diff
@@ -163,10 +165,11 @@ def quantity(self):
         """
         return u.Quantity(self)
 
-class Projection(LowerDimensionalObject):
+class Projection(LowerDimensionalObject, SpatialCoordMixinClass):
```

Comment:
What if you swapped the order of the inheritance?  Maybe that would fix the doc issues?